config/initializers/settings_digests.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+Rails.configuration.after_initialize do
+  if ApplicationRecord.connection.database_exists? && Setting.table_exists?
+    digest = Digest::SHA256.hexdigest(Setting.custom_css.to_s)
+    Rails.application.config.x.settings_digests[:custom_css] = digest
```

Comment:
Wouldn't you have to restart Rails after updating custom CSS in the admin settings for them to take effect?

Reply:
There's another update in `Form::AdminSettings` as well. So - it gets set here once on app launch, and then re-set if/when the value is updated from there.

Noted a little in description, but that could potentially be made cleaner if we want either a) move to `Setting` and use an actual `after_` callback (its an AR class) b) add ActiveModel::Callbacks to Form::AS.

Reply:
But this will only work on the same Rails node as where the form is processed?

Reply:
Ah yes, probably ... they'd all be fine on launch but once an update happened the non-update-processing nodes would have the old value and not know to expire.

Will rework this with something shared (rails cache, or back to a query if we can share an existing one). Might explore the "settings loading consolidation" idea from description here as well first.